### PR TITLE
Ruby: Release 1.0.0-alpha.10 and 1.0.0-alpha.11

### PR DESCRIPTION
**Pull request by mkllnk**
Here are all the latest updates of the Ruby Connector.

I decided not to wait for the next branch of the data model to be merged because we need these changes now.

### CODE REVIEW

**Comment by lecoqlibre** [comment]
@mkllnk Should I wait the new Semantizer version before merging it? Do you want to change the requirements for Semantizer in the gemspec?

**Comment by mkllnk** [comment]
> Should I wait the new Semantizer version before merging it? Do you want to change the requirements for Semantizer in the gemspec?

Okay, let's wait for that. I'll do a new release and update this PR after the semantizer has been released.

**Comment by lecoqlibre** [comment]
@mkllnk The Semantizer 1.1.1 has been released, can you update this PR accordingly?

**Comment by mkllnk** [comment]
> The Semantizer 1.1.1 has been released, can you update this PR accordingly?

Done.

File: src/org/datafoodconsortium/connector/codegen/ruby/static/connector.gemspec [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+  s.license     = "MIT"
+
+  s.metadata = {
+    "changelog_uri" =>
+      "https://github.com/datafoodconsortium/connector-ruby/blob/main/CHANGELOG.md",
+    "source_code_uri" =>
+      "https://github.com/datafoodconsortium/connector-ruby/",
+  }
 
   s.add_runtime_dependency "virtual_assembly-semantizer", "~> 1.0", ">= 1.0.5"
```
[the remark this answers is not in the document]
Comment (lecoqlibre) [reply src/org/datafoodconsortium/connector/codegen/ruby/static/connector.gemspec:19]: OK som I'm merging this into branch v1.1.0.

**Pull request merged**